src/ol_infrastructure/applications/airbyte/__main__.py

```diff
@@ -294,12 +301,6 @@
     role=airbyte_server_instance_role.name,
 )
 
-airbyte_server_instance_profile = iam.InstanceProfile(
-    f"airbyte-server-instance-profile-{env_name}",
-    role=airbyte_server_instance_role.name,
-    path="/ol-infrastructure/airbyte-server/profile/",
-)
-
```

Comment:
Where does the IAM policy get bound to the service account?

Reply:
Haven't done that bit yet.